blogposts/2020/3.18/custom-search-pages.md

```diff
@@ -0,0 +1,34 @@
+title: 'Custom search pages on Sourcegraph Cloud'
+author: 'Farhan'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: custom-search-pages
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+## Custom search pages on Sourcegraph Cloud (#[11526](https://github.com/sourcegraph/sourcegraph/issues/11526))
```

Comment:
Title should be an h1

```suggestion
# Custom search pages on Sourcegraph Cloud (#[11526](https://github.com/sourcegraph/sourcegraph/issues/11526))
```

Reply:
Oh, yes Herzog did upload to YT for me! I forgot to take it out of the folder, I'll do that now. And I'll also look at the bucket thing